kuma/static/js/redesign.js

```diff
@@ -0,0 +1,28 @@
+(function ($) {
+    'use strict';
+
+    var redesignNotice = document.getElementById('redesignNotice');
+
+    if(!redesignNotice) {
+        return;
+    } else {
+        if(mdn.features.localStorage) {
+            // first visit to redesign?
+            var seenNotice = localStorage.getItem('redesignNotice') === 'true' ? true : false;
+            if (!seenNotice) {
+                var $redesignNotice = $(redesignNotice);
+
+                // wire close button
+                $('#redesignHide').on('click', function() {
+                    $redesignNotice.remove();
+                });
+                // show message
+                $redesignNotice.prependTo($('#content'));
```

Comment:
JS is magic 🎩 ✨. I had some problems getting this to work on the homepage, but it appears to be related to deleting ``redesignNotice`` and successfully refreshing.